src/issue/screens/issue.screen.js

```diff
@@ -40,7 +40,8 @@ const mapStateToProps = state => ({
 });
 
 const mapDispatchToProps = dispatch => ({
-  getIssueCommentsByDispatch: url => dispatch(getIssueComments(url)),
+  getIssueCommentsByDispatch: issueCommentsURL =>
+    dispatch(getIssueComments(issueCommentsURL)),
```

Comment:
Might be nice to keep this as `url => dispatch(getIssueComment(url)),` to keep things on one line, but either way works for me